jquery.scrollify.js

```diff
@@ -58,7 +58,6 @@ if touchScroll is false - update index
     timeoutId2,
     $window = $(window),
     portHeight,
-    top = $window.scrollTop(),

```

Comment:
removing this line means `top` is never defined, but it's still being used later in the code.

Reply:
I defined `top` below the settings-part in line 95.
top = $(settings.target).scrollTop();

Reply:
https://github.com/lukehaas/Scrollify/blob/6092bb2648fef5d024cb7c90717e05402358914b/jquery.scrollify.js#L95

Reply:
This is just assigning a value to an undeclared variable, it's not valid.

Reply:
If you look at line 49, you can see all the variables are declared from this line.

Reply:
I have a comma at the end of line 94 - not a semicolon.
Doesn't that mean that `top` gets declared with (and after) all other variables starting from line 49?

See:
https://github.com/lukehaas/Scrollify/blob/6092bb2648fef5d024cb7c90717e05402358914b/jquery.scrollify.js#L49-L95

I just moved the `top` declaration after `settings` declaration.

Reply:
Ah, I missed that it had been moved. That's all good then.